rego/Groups.rego

```diff
@@ -44,8 +44,8 @@ tests contains {
     "NoSuchEvent": true
 }
 if {
-    DefaultSafe := false
-    Events := utils.FilterEventsOU(LogEvents, "GroupsSharingSettingsProto collaboration_policy", utils.TopLevelOU)
+    DefaultSafe := true
+    Events := utils.FilterEvents(LogEvents, "GroupsSharingSettingsProto collaboration_policy", utils.TopLevelOU)
```

Comment:
Why did this line change from `FilterEventsOU()` to `FilterEvents()`? The `FilterEvents()` function is (unofficially) deprecated.

Reply:
must have been a typo, reverted it